torchvision/prototype/models/optical_flow/raft.py

```diff
@@ -23,16 +21,27 @@
 
 
 class Raft_Large_Weights(WeightsEnum):
-    pass
-    # C_T_V1 = Weights(
-    #     # Chairs + Things
-    #     url="",
-    #     transforms=RaftEval,
-    #     meta={
-    #         "recipe": "",
-    #         "epe": -1234,
-    #     },
-    # )
+    C_T_V1 = Weights(
+        # Chairs + Things, ported from original paper repo (raft-things.pth)
+        url="https://download.pytorch.org/models/raft_large_C_T_V1-22a6c225.pth",
+        transforms=RaftEval,
+        meta={
+            "recipe": "https://github.com/princeton-vl/RAFT",
+            "sintel_train_cleanpass_epe": 1.4411,
```

Comment:
Does it make sense to rename one of them as the default `epe`? This will allow you to add the metric in the schema of meta-data for optical flow models. It's also worth considering introducing a dictionary entry in the meta-data that holds other `epe` values for for different datasets etc.

Reply:
> Does it make sense to rename one of them as the default epe?

Unfortunately no, because the rest of the weights will be trained on sintel, so reporting the epe on the trainset would not be relevant

Reply:
I'm happy to have a dict or something else to properly keep track of the other metrics though - ultimately I think it would make sense to also have 1px, 3px etc. I think we'll have a better idea of what it should look like once the rest of the weights are available

Reply:
Sounds good, no strong opinions. You could dump all the metrics in an `epe` dictionary. Then you would be able to include this on the schema. Up to you.